src/Core/src/Handlers/Entry/EntryHandler.Android.cs

```diff
@@ -148,14 +146,6 @@ static void MapFocus(IEntryHandler handler, IEntry entry, object? args)
 				handler.PlatformView.Focus(request);
 		}
 
-		void OnViewAttachedToWindow(object? sender, ViewAttachedToWindowEventArgs e)
```

Comment:
What was this doing that we don't want anymore?

Reply:
We added it here https://github.com/dotnet/maui/commit/55ed1ea28148f1ae4642873be51fc2cd094a4ac9 to fix an issue related with the `ReturnType` property. Verified, is no longer necessary.